.pre-commit-config.yaml

```
@@ -11,3 +11,7 @@ repos:
     rev: "v2.7.1"
     hooks:
       - id: prettier
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v3.15.0
+    hooks:
+    -   id: pyupgrade
```

We should add the Python version we want to upgrade to as explained here: https://pre-commit.com/#passing-arguments-to-hooks and https://pre-commit.com/#pre-commit-autoupdate

```suggestion
    -   id: pyupgrade
        args: [--py38-plus]
```